src/components/SingleList.jsx

```diff
@@ -1,12 +1,18 @@
 import './SingleList.css';
+import { useStateWithStorage } from '../utils';
+
+export function SingleList({ name, index, path }) {
+	const [listPath, setListPath] = useStateWithStorage(
```

Comment:
If this is already used in App.jsx, does it need to be initialized again here? I think setListPath can be passed down from Home into this component. We tested it and it seemed to work okay.

Reply:
Can you show me what you did while testing to get it to work without initializing again in the SingleList component?

Reply:
You're right! Got it working after some tinkering.